sound/soc/sof/pcm.c

```diff
@@ -619,6 +619,27 @@ static int sof_pcm_new(struct snd_soc_component *component,
 	return 0;
 }
 
+static void ssp_dai_config_pcm_params_match(struct snd_sof_dev *sdev, const char* name,
+					    struct snd_pcm_hw_params *params)
+{
+	struct sof_ipc_dai_config *config;
+	struct snd_sof_dai *dai;
+	int i;
+
+	/* we might have several dai's associated with same link, need to loop all */
+	list_for_each_entry(dai, &sdev->dai_list, list) {
+		if (!dai->name || strcmp(name, dai->name))
+			continue;
+		for (i = 0; i < dai->number_configs; i++) {
+			config = &dai->dai_config[i];
+			if (config->ssp.fsync_rate == params_rate(params)) {
+				dev_dbg(sdev->dev, "DAI config pcm params match at index %d\n", i);
```

Comment:
not sure this is a very helpful log. If anything index would confuse me with DAI index.

Reply:
ok, can try to make this more readable, anyway the dbg msg here is _really_ useful